environments/sbox.tfvars

```diff
@@ -21,4 +20,22 @@ frontends = [
     certificate_name = "STAR-sandbox-platform-hmcts-net"
     disabled_rules   = {}
   },
-]
\ No newline at end of file
+  {
+    name             = "idam-web-public"
+    custom_domain    = "idam-web-public-aks.sandbox.platform.hmcts.net"
+    backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
+    certificate_name = "STAR-sandbox-platform-hmcts-net"
+    disabled_rules = {
```

Comment:
would be best to start with no rules / add some sensible exclusions and then build up as you go

Reply:
@timja these rules are the ones we previously had approved from SecOps. No impact assessment has been made to see what the consequence of removing these now would be. If possible we'll try and do that for the 2.0.0 but we have not got a lot of time left.

Reply:
@timja I'll go ahead and remove these and see what impact there is for our tests. They might need to come back in.